.travis.yml

```diff
@@ -12,6 +12,10 @@ install:
   - source activate test-env
   - pip install https://github.com/qiime2/qiime2/archive/master.zip https://github.com/qiime2/q2-types/archive/master.zip
   - pip install .
```

Comment:
Add this line after `pip install .`:

`- 'echo "backend: Agg" > matplotlibrc'`

This should fix the travis error